### PR TITLE
updated version number and quality level

### PR DESCRIPTION
**Pull request by joespeed**
@eboasson updated version number 1.0.0 and quality level 3

### CODE REVIEW

**Comment by joespeed** [comment]
changed to QL3 per @eboasson because declared dependency on rosidl package that lacks documentation

**Review by clalancette — APPROVED** [review]
Looks good to me.  I'll leave this to the maintainers to merge.

**Pull request merged**
